tests/test_utils/test_path_helpers.py

```diff
@@ -0,0 +1,196 @@
+
+
+def test_create_staging_config(config, expected_staging_dict):
+    """Test create_staging_config function."""
+
+    staging_dict = create_staging_config(config)
+
+    assert staging_dict == expected_staging_dict
+
+
+def test_create_ni_staging_config(config):
+    """Test create_ni_staging_config function."""
+
+    expected_ni_staging_dict = {
+        "ni_full_responses": (
+            "R:/DAP_emulation/2022_surveys/BERD/03_northern_ireland/2021/TEST_ni.csv"
+        ),
+        "ni_staging_output_path": (
+            "R:/DAP_emulation/2022_surveys/BERD/03_northern_ireland/ni_staging_qa"
+        ),
+    }
+    ni_staging_dict = create_ni_staging_config(config)
+
+    assert ni_staging_dict == expected_ni_staging_dict
+
+
+def test_create_mapping_config(config):
+    """Test create_mapping_config function."""
+
+    expected_mapping_dict = {
+        "postcodes_mapper": "R:/DAP_emulation/2022_surveys/mappers/v1/pcodes_2022.csv",
+        "itl_mapper_path": "R:/DAP_emulation/2022_surveys/mappers/v1/itl_2022.csv",
+    }
+    mapping_dict = create_mapping_config(config)
+
+    assert mapping_dict == expected_mapping_dict
+
+
+def test_create_module_config_imputation_case(config):
+    """Test create_module_config function for the imputation module."""
+
+    expected_imputation_dict = {
+        "qa_path": "R:/DAP_emulation/2022_surveys/BERD/06_imputation/imputation_qa",
+        "manual_trimming_path": (
+            "R:/DAP_emulation/2022_surveys/BERD/06_imputation/manual_trimming"
+        ),
+    }
+    imputation_dict = create_module_config(config, "imputation")
+
+    assert imputation_dict == expected_imputation_dict
+
+
+@pytest.fixture(scope="module")
+def expected_outliers_dict():
+    expected_outliers_dict = {
+        "qa_path": "R:/DAP_emulation/2022_surveys/BERD/07_outliers/outliers_qa",
+        "auto_outliers_path": (
+            "R:/DAP_emulation/2022_surveys/BERD/07_outliers/auto_outliers"
+        ),
+    }
+    return expected_outliers_dict
+
+
+def test_create_module_config_outliers_case(config, expected_outliers_dict):
+    """Test create_module_config function for the outliers module."""
+
+    outliers_dict = create_module_config(config, "outliers")
+
+    assert outliers_dict == expected_outliers_dict
+
+
+def test_update_config_with_paths(
+    config, expected_staging_dict, expected_outliers_dict
+):
+    """Test update_config_with_paths function."""
+    updated_config = update_config_with_paths(config)
+
+    assert "staging_paths" in updated_config
+    assert "ni_paths" in updated_config
+    assert "mapping_paths" in updated_config
+
+    for module_name in [
+        "construction",
+        "imputation",
+        "outliers",
+        "estimation",
+        "apportionment",
+        "outputs",
+    ]:
+        assert f"{module_name}_paths" in updated_config
+
+    assert updated_config["staging_paths"] == expected_staging_dict
+    assert updated_config["outliers_paths"] == expected_outliers_dict
```

Comment:
Are you able to add assert messages to each of the asserts in this test module? It makes it easier to debug in the event of an error and is good practice.